src/app/app.module.ts

```diff
@@ -31,6 +31,9 @@ import {APP_LOCALE_ID} from '../environments/app-locale';
 import {ShareButtonsModule} from '@ngx-share/buttons';
 import {FaIconLibrary} from '@fortawesome/angular-fontawesome';
 import {shareButtonsIcons} from '@/icons';
+import {registerLocaleData} from '@angular/common';
```

Comment:
remove these lines, this will break a build

Reply:
okay, i will, sorry. i have no idea how they got in there again after i removed them last time. let me know if it's ok this time.